.github/workflows/build.yml

```diff
@@ -212,13 +222,74 @@ jobs:
           npm run test:generate
           npm run test --shard=${{ matrix.shard }}
 
+  console:
+    name: E2E Console
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install Dependencies
+        uses: bahmutov/npm-install@v1
+
+      - name: Download Dist Files
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+
+      - name: Link Wing CLI
+        run: npm i -g ./apps/wing
+
+      - name: Check Wing Version
+        run: wing --version
```

Comment:
Does the Console use the global version of the cli (when running the tests in `pnpm run build`)? Could it use the linked one instead?